synaptic_reconstruction/tools/volume_reader.py

```diff
@@ -22,7 +23,17 @@ def get_reader(path: PathOrPaths) -> Optional[ReaderFunction]:
 def _read_mrc(path, fname):
     with open_file(path, mode="r") as f:
         data = f["data"][:]
-    layer_attributes = {"name": fname, "colormap": "gray"}
+    
+    metadata = {
+        "file_path": path
+    }
+    read_voxel_size(path, metadata)
```

Comment:
Implicitly writing the `voxel_size` to the function here is a bit confusing.
It would be much more straightforward to do something like this:
```python
voxel_size = read_voxel_size(path)  # Return None if some exception happens
metadata = {"file_path": path, "voxel_size": voxel_size}
```